docker-app/qfieldcloud/core/serializers.py

```diff
@@ -14,7 +14,10 @@
     ProjectCollaborator,
     Team,
     User,
+    Person,
+    TeamMember
```

Comment:
```suggestion
    TeamMember,
```

Do you have pre-commit, this should have a trailing comma.

Reply:
pre-commit applied!